File: app/src/main/java/gcm/heruijun/com/travelflangcm/SplashActivity.java
package gcm.heruijun.com.travelflangcm;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import com.gcm.heruijun.base.ui.activity.BaseCompatActivity;
import com.gcm.heruijun.base.utils.AppUtils;

/**
 * Created by heruijun on 2017/12/10.
 */

public class SplashActivity extends BaseCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        AppUtils.getHandler().postDelayed(() -> {
            startActivity(new Intent(SplashActivity.this, MainActivity.class));
            finish();
        }, 2000);
    }
}

File: app/src/main/java/gcm/heruijun/com/travelflangcm/SplashActivity.kt
package gcm.heruijun.com.travelflangcm

import android.os.Bundle

import com.gcm.heruijun.base.ui.activity.BaseCompatActivity
import com.gcm.heruijun.base.utils.AppUtils
import org.jetbrains.anko.*

/**
 * Created by heruijun on 2017/12/10.
 */

class SplashActivity : BaseCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        AppUtils.handler.postDelayed({
            startActivity(intentFor<MainActivity>())
            finish()
        }, 2000)
    }
}
